src/main/java/Listing.java

```diff
@@ -18,17 +18,21 @@ public static void main(String[] args) {
 
     private static void processListing(){
         Integer chooseOption;
-        System.out.print("choose one of the option given below:" + "\n" + "1.Buy" + "\n" + "2.Sell" +"\n");
+        System.out.print("choose one of the option given below:" + "\n" + "1.Buy" + "\n" + "2.Sell" +"\n" + "3.View existing property details" + "\n");
         System.out.print("Enter your option:");
         Scanner scanner = new Scanner(System.in);
         chooseOption = scanner.nextInt();
         HouseDetails houseDetails = new HouseDetails();
+        SellPropertyService sellPropertyService = new SellPropertyService();
         if(chooseOption == 1){
-            SellPropertyService sellPropertyService = new SellPropertyService();
             sellPropertyService.processSellProperty();
-        }else {
+        }else if (chooseOption == 3 ){
+            sellPropertyService.readAFile(sellPropertyService.getInputFromUserToReadExistingProperty());
+            }
+        else{
```

Comment:
Looks like `This feature is under construction. Please come back soon.` is common for option 1 and option greater than 3. Ideally it would be correct for option 1 but for other invalid options, we can say bad option selected.